repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -140,8 +144,10 @@ def scan_events(path):
     try:
         events = parse_file(path)
     except (ValueError, KeyError) as error:
-        raise StopActorExecutionError(
-            message='Found invalid or outdated PES data.',
-            details={'details': error, 'hint': 'Consult docs to retrieve updated data.'})
+        title = 'Missing/Invalid PES data file ({})'.format(path)
+        summary = 'Read documentation at: https://access.redhat.com/articles/3664871 for more information ' \
+            'about how to retrieve the files'
+        reporting.report_generic(title=title, summary=summary, severity='high', flags=['inhibitor'])
+        raise StopActorExecutionError(message=title, details={'hint': summary, 'details': str(error)})
```

Comment:
This doesn't make sense. You cannot report when you tell that execution should be stopped (so the executor taking care of report will not be procesesd). Only the first one or the second one.

Reply:
@pirat89 I opened another PR to solve this https://github.com/oamg/leapp-repository/pull/109